RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -45,6 +45,33 @@
 akPu6PFJets = akPu5PFJets.clone(rParam       = cms.double(0.6), puPtMin = 30)
 akPu7PFJets = akPu5PFJets.clone(rParam       = cms.double(0.7), puPtMin = 35)
 
+hiPFCandCleanerforJets = cms.EDFilter('GenericPFCandidateSelector',
+                                src = cms.InputTag('particleFlow'),
+                                cut = cms.string("pt>5 && abs(eta)< 2")
+                                )
+
+ak4PFJetsForFlow = akPu5PFJets.clone(
+   Ghost_EtaMax = 5.0,
+   Rho_EtaMax = 4.4,
+   doRhoFastjet = False,
+   jetPtMin = 15.0,
+   nSigmaPU = cms.double(1.0),
+   rParam = 0.4,
+   radiusPU = cms.double(0.5),
+   src = "hiPFCandCleanerforJets",
+)
+
+# ak4PFJetsForFlow = akPu5PFJets.clone(
+#     Ghost_EtaMax = cms.double(5.0),
+#     Rho_EtaMax = cms.double(4.4),
+#     doRhoFastjet = cms.bool(False),
+#     jetPtMin = cms.double(15.0),
+#     nSigmaPU = cms.double(1.0),
+#     rParam = cms.double(0.4),
+#     radiusPU = cms.double(0.5),
+#     src = cms.InputTag("hiPFCandCleanerforJets"),
+# )
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant